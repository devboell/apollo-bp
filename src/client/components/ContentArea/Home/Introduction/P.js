import styled from 'styled-components'

const P = styled.p`
  text-indent: 50px;
`

export default P
