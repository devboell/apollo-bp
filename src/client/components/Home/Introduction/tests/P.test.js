import React from 'react'
import 'jest-styled-components'

import P from '../P'

it('P, styled component snapshot', () => {
  const wrapper = shallow(<P />)
  expect(wrapper).toMatchStyledComponentsSnapshot()
})
