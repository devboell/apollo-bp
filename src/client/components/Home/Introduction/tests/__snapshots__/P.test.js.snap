// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`P, styled component snapshot 1`] = `
.c0 {
  -webkit-text-indent: 50px;
  text-indent: 50px;
}

<p
  className="c0"
/>
`;
